backend/dataall/modules/catalog/api/queries.py

```diff
@@ -21,7 +21,7 @@
 
 SearchGlossary = gql.QueryField(
     name='searchGlossary',
-    doc='Search glossary ',
+    description='Search glossary ',
```

Comment:
Here, after the transformation described above, we will have both space and \n.
Is it ok?

Reply:
not sure I follow but the extra space is not a problem - will have a later PR that adds descriptions to all of our GQL queries, mutations, etc. and can clean up this description then